modules/apps/frontend-js/frontend-js-web/test/liferay/delegate/delegate.es.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import delegate from '../../../src/main/resources/META-INF/resources/liferay/delegate/delegate.es';
+import {getByTestId} from '@testing-library/dom';
+import userEvent from '@testing-library/user-event';
+
+describe('delegate', function () {
+	afterEach(() => {
+		document.body.innerHTML = '';
+	});
+
+	it('should trigger delegate listener for matched elements', function () {
```

Comment:
I think [the linter is going to warn you](https://github.com/liferay/liferay-frontend-projects/blob/master/projects/eslint-config/plugins/liferay/docs/rules/no-it-should.md) about these "should".